definitions/tools/interval_list_expand.cwl

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env cwl-runner
+
+cwlVersion: v1.0
+class: CommandLineTool
+label: "expand interval list regions by a given number of basepairs"
+
+baseCommand: ["/usr/bin/java", "-Xmx16g", "-jar", "/usr/picard/picard.jar", "IntervalListTools"]
+
+arguments:
+    ["OUTPUT=", { valueFrom: $(runtime.outdir)/expanded.interval_list }, "UNIQUE=TRUE"]
+requirements:
+    - class: ResourceRequirement
+      ramMin: 16000
+    - class: DockerRequirement
+      dockerPull: "mgibio/picard-cwl:2.18.1"
+inputs:
+    interval_list:
+        type: File
+        inputBinding:
+            prefix: "INPUT="
```

Comment:
Should we go with `separate: false` on these inputs?  (I think Picard is up to the task of dealing with it either way, but that'd be the expected way to do it.)  Alternatively we could upgrade to the GATK4 version, which lets us specify these a different way, like `--INPUT`.